cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -353,8 +353,9 @@ data TxOut ctx era = TxOut (AddressInEra    era)
                            (TxOutDatum ctx  era)
                            (ReferenceScript era)
 
-deriving instance Eq   (TxOut ctx era)
-deriving instance Show (TxOut ctx era)
+deriving instance Eq      (TxOut ctx era)
+deriving instance Generic (TxOut ctx era)
```

Comment:
What's this generic instance for?

Reply:
It was previously used, but no longer.  Removed the generic instance.

Reply:
Yep, you're right.  This instance is unused.